### PR TITLE
Update npm dependencies

### PR DESCRIPTION
**Pull request by danielcooperxyz**
**Problem**
The dependencies should be updated.

**Solution**
Update the dependencies.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -29,29 +29,29 @@
...
     "compression": "^1.7.4",
     "cors": "^2.8.5",
-    "dotenv": "^8.2.0",
-    "envalid": "^7.0.0",
-    "express": "^4.17.1",
-    "express-openapi": "^10.0.0",
-    "pino": "^6.11.1",
-    "pino-http": "^5.5.0",
-    "swagger-ui-express": "^4.2.0"
+    "dotenv": "^16.0.0",
```
Comment (dblane-digicatapult) [inline package.json:35]: Its probably worth noting this here
https://github.com/motdotla/dotenv/blob/master/CHANGELOG.md

However as this is a fresh repository there is no expectation that there are .env files currently existing.

File: package.json [hunk 2]
```diff
@@ -29,29 +29,29 @@
...
+    "envalid": "^7.3.1",
+    "express": "^4.17.3",
+    "express-openapi": "^10.1.0",
+    "pino": "^7.10.0",
```
Comment (dblane-digicatapult) [inline package.json:39]: @danielcooperxyz Can you please verify that the breaking changes don't affect anything in this repository?
https://github.com/pinojs/pino/releases/tag/v7.0.0

File: package.json [hunk 3]
```diff
@@ -29,29 +29,29 @@
...
+    "chai": "^4.3.6",
     "chai-json": "^1.0.0",
-    "depcheck": "^1.4.0",
-    "eslint": "^7.21.0",
-    "eslint-config-prettier": "^8.1.0",
-    "eslint-plugin-prettier": "^3.3.1",
-    "mocha": "^9.1.2",
-    "nodemon": "^2.0.7",
+    "depcheck": "^1.4.3",
+    "eslint": "^8.13.0",
```
Comment (dblane-digicatapult) [inline package.json:47]: Please verify we comply with the new changes
https://eslint.org/docs/8.0.0/user-guide/migrating-to-8.0.0

**Review by dblane-digicatapult — APPROVED** [review]

File: package.json (hunk 2 above)
Comment (danielcooperxyz) [reply package.json:39]: Looks like there have been some changes but nothing that breaks or changes the behaviour of our current usage.

File: package.json (hunk 3 above)
Comment (danielcooperxyz) [reply package.json:47]: Looks good to me!

**Pull request merged**